### DISCUSSION TITLE
Thanks for following me!

### DISCUSSION THREAD

**Discussion by YURLAK** [esin/esin#11] [Guestbook]
Hi!Excuse me I'm a student.But I'm pleasure to discuss programming with you!I'm interesting about Python and Rust.[My website](https://yurlak.github.io)

**Comment by neoheartbeats** [comment]
Following @YURLAK . Me too. Sorry for my poor English (Not a native speaker). I am also college student. Now focusing on building an Emacs program for researchers (just like [scimax](https://github.com/jkitchin/scimax) but considered adding some new ideas). Pleasure for discussing! ;)

---

BTW, to @YURLAK, I am also Chinese. Nice to meet you. You can contact me via my Discord username `ilyaw39#0120`
